data/data-pipeline/data_pipeline/etl/sources/maryland_ejscreen/etl.py

```diff
@@ -0,0 +1,115 @@
+    Please see the README in this module for further details.
+    """
+
+    def __init__(self):
+        self.MARYLAND_EJSCREEN_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL + "/MD_EJScreen.zip"
+        )
+
+        self.SHAPE_FILES_PATH = self.TMP_PATH / "mdejscreen"
+        self.OUTPUT_CSV_PATH = self.DATA_PATH / "dataset" / "maryland_ejscreen"
+
+        self.COLUMNS_TO_KEEP = [
+            self.GEOID_TRACT_FIELD_NAME,
+            field_names.MARYLAND_EJSCREEN_TRACT_25_PERCENT_FIELD,
+            field_names.MARYLAND_EJSCREEN_TRACT_50_PERCENT_FIELD,
+            field_names.MARYLAND_EJSCREEN_TRACT_75_PERCENT_FIELD,
+            field_names.MARYLAND_EJSCREEN_TRACT_90_PERCENT_FIELD,
+            field_names.MARYLAND_PERCENTILE_FIELD_NAME,
+        ]
+
+        self.df: pd.DataFrame
+
+    def extract(self) -> None:
+        logger.info("Downloading Maryland EJSCREEN Data")
+        super().extract(
+            self.MARYLAND_EJSCREEN_URL,
+            self.TMP_PATH,
+        )
+
+    def transform(self) -> None:
+        logger.info("Transforming Maryland EJSCREEN Data")
+
+        list_of_files = list(glob(str(self.SHAPE_FILES_PATH) + "/*.shp"))
+
+        # Ignore counties becauses this is not the level of measurement
+        # that is consistent with our current scoring and ranking methodology.
+        dfs_list = [
+            gpd.read_file(f)
+            for f in list_of_files
+            if not f.endswith("CountiesEJScore.shp")
+        ]
+
+        # Set the Census tract as the index and drop the geometry column
+        # that produces the census tract boundaries.
+        # The latter is because Geopandas raises an exception if there
+        # are duplicate geometry columns.
+        # Moreover, since the unit of measurement is at the tract level
+        # we can consistantly merge this with other datasets
+        dfs_list = [
+            df.set_index("Census_Tra").drop("geometry", axis=1)
+            for df in dfs_list
+        ]
+        # pylint: disable=unsubscriptable-object
+        combined_df = gpd.GeoDataFrame(pd.concat(dfs_list, axis=1))
+
+        # Reset index so that we no longer have the tract as our index
+        combined_df = combined_df.reset_index()
+        # coerce into integer into
+        # pylint: disable=unsupported-assignment-operation, unsubscriptable-object
+        combined_df["Census_Tra"] = (combined_df["Census_Tra"]).astype(int)
+
+        # Drop the 10 census tracts that are zero: please see here:
+        # https://github.com/usds/justice40-tool/issues/239#issuecomment-995821572
+        combined_df = combined_df[combined_df["Census_Tra"] != 0]
+
+        # Set our class instance variable.
+        self.df = combined_df.copy()
+
+        # Rename
+        self.df.rename(
+            columns={
+                "Census_Tra": self.GEOID_TRACT_FIELD_NAME,
+                "EJScore": field_names.MARYLAND_PERCENTILE_FIELD_NAME,
```

Comment:
If I'm reading the documentation and data correctly, this is one of those very small mistakes that has massive implications for our data analysis. I'm pretty sure the EJScore is *not* a percentile – for instance, 

> The two "Pollution Burden" average scores are then averaged together and the result is multiplied by the average of the "Population Characteristics" categories to get the total EJ Score for each tract. 

This is somewhat obvious from looking at the data and the results -- only 2% of Maryland's tracts are > .90 on the EJScore. If it was a percentile, that would be 10% of Maryland's tracts by definition. 

If you agree with this analysis, could you update and re-run the statistical output? Thanks!

Reply:
Hi @lucasmbrown-usds - I am not clear nor do I completely agree with the statements outlined here. Citing [this](https://p1.cgis.umd.edu/mdejscreen/help.html) and below from the documentation, I think there may be a misunderstanding? I am happy to clarify and address your specific concerns. 

![Screen Shot 2021-12-31 at 1 01 14 PM](https://user-images.githubusercontent.com/94847739/147834977-6bde0539-43bf-4c1c-a66d-3d69f3e172fd.png)

Reply:
Or, alternatively, if we get the sense that Maryland is prioritizing *not* based on the percentile but based on the EJScore (raw) itself, that's fine. We can rename these fields to remove all references to percentiles (since they're not percentiles) choose an appropriate cutoff for the comparison tool. Maybe we re-run the comp tool with MARYLAND_EJSCREEN_TRACT_75_PERCENT_FIELD. Thanks!